src/roiextractors/extractors/minian/miniansegmentationextractor.py

```diff
@@ -0,0 +1,189 @@
+"""A SegmentationExtractor for Minian.
+
+Classes
+-------
+MinianSegmentationExtractor
+    A class for extracting segmentation from Minian output.
+"""
+
+from pathlib import Path
+
+import zarr
+import warnings
+import numpy as np
+import pandas as pd
+
+from ...extraction_tools import PathType
+from ...segmentationextractor import SegmentationExtractor
+
+
+class MinianSegmentationExtractor(SegmentationExtractor):
+    """A SegmentationExtractor for Minian.
+
+    This class inherits from the SegmentationExtractor class, having all
+    its functionality specifically applied to the dataset output from
+    the 'Minian' ROI segmentation method.
+    """
+
+    extractor_name = "MinianSegmentation"
+    is_writable = True
+    mode = "file"
+
+    def __init__(self, folder_path: PathType):
+        """Initialize a MinianSegmentationExtractor instance.
+
+        Parameters
+        ----------
+        folder_path: str
+            The location of the folder containing minian .zarr output.
+        """
+        SegmentationExtractor.__init__(self)
+        self.folder_path = folder_path
+        self._roi_response_denoised = self._trace_extractor_read(field="C")
+        self._roi_response_baseline = self._trace_extractor_read(field="b0")
+        self._roi_response_neuropil = self._trace_extractor_read(field="f")
```

Comment:
Let's say first that the background can contain neuropil signal. The problem is that I wanted to map this "f" signal with something already in the SegmentationExtractor.

Now, I am not super happy with this formulation I think we should have two separate variables:

1. _roi_response_neuropil with the respective _neuropil_images_masks. This can be used to store the actual neuropil signals that sometimes refer to the area surrounding the ROIs
2. _roi_response_background with the respective _background_images_masks. This can be used more generally to store the signal in the field of view that excludes the areas in the identified ROIs (as in this case).

I will open an issue on roiextractor to discuss this with the others.

Reply:
OK, makes sense.